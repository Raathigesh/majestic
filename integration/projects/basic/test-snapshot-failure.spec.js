import renderer from 'react-test-renderer';
import React from 'react';
import App from './app';

describe('test', () => {
  it('Snapsh0t test', () => {
    // Make sure we don't use 'snapshot' because it fools the snapshot button
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
