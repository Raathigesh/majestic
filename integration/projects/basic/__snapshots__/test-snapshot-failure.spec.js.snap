// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Snapsh0t test 1`] = `
<div
  className="App"
>
  Hello world
</div>
`;
